src/examples/pBanach.lean

```diff
@@ -17,8 +17,7 @@ variables (p : ℝ≥0) (V : pBanach.{0} p) (S : Profinite.{0})
 
 section pBanach
 
-/-! The $p$-Banach space $V$ is a complete and separated topological $ℝ$-vector space. -/
-
+/-! The $p$-Banach space $V$ is a complete and separated topological `ℝ`-vector space. -/
```

Comment:
This is now mixing `$` and `` ` ``. Is that intentional?

Reply:
I changed all the `$` to `\`` in this file.